routing/routing_benchmarks/bicycle_routing_tests.cpp

```diff
@@ -45,7 +45,7 @@ UNIT_CLASS_TEST(BicycleTest, Smoke)
   TestRouters(start, final);
 }
 
-UNIT_CLASS_TEST(BicycleTest, MailRuOfficeToUlSimanova5K2)
```

Comment:
As I wrote answering your comment in 4300 now we use in routing_integration_tests test names with full address: <Country><City><Street>... . Doesn't it give us less info than <Country><Cicy>_TestXX?